src/main/java/io/kestra/plugin/git/AbstractGitTask.java

```diff
@@ -487,8 +487,12 @@ public static DiffLine deletedKestra(String file, String key, Kind kind) {
 
         @SneakyThrows
         public static URI writeIonFile(RunContext runContext, List<DiffLine> diffs) {
-            byte[] ionContent = JacksonMapper.ofIon().writeValueAsBytes(diffs);
-            try (ByteArrayInputStream input = new ByteArrayInputStream(ionContent)) {
+            List<DiffLine> changed = diffs.stream()
+                .filter(d -> d.getAction() != Action.UNCHANGED)
+                .toList();
```

Comment:
Maybe put this in a method because used also in `NamespaceSync` and in `TenantSync` ⬇️ 

```java
diffs.stream()
                .filter(d -> d.getAction() != Action.UNCHANGED)
                .toList()
```